SasquatchMac/SasquatchMacObjC/AppDelegate.h

```diff
@@ -1,8 +1,10 @@
 #import <Cocoa/Cocoa.h>
+#import <CoreLocation/CoreLocation.h>
 
 @import AppCenterCrashes;
 @import AppCenterPush;
+@import CoreLocation;
```

Comment:
What is this for?

Reply:
yeah, when I just imported at line 2, I found that the intelligence cannot show me the code completion in AppDelegate.m file, so I made this additional import to try... the intelligence issue maybe just because of build/reference issue, I will try this out again today. Thanks~